Tests/test_lib_pack.py

```diff
@@ -280,8 +280,14 @@ def test_RGBA(self):
         self.assert_unpack("RGBA", "RGBa", 4,
             (63, 127, 191, 4), (159, 191, 223, 8), (191, 212, 233, 12))
         self.assert_unpack("RGBA", "RGBa",
-            b'\x01\x02\x03\x00\x10\x20\x30\xff',
-            (0, 0, 0, 0), (16, 32, 48, 255))
+            b'\x01\x02\x03\x00\x10\x20\x30\x7f\x10\x20\x30\xff',
+            (0, 0, 0, 0), (32, 64, 96, 127), (16, 32, 48, 255))
+        self.assert_unpack("RGBA", "RGBaX",
+            b'\x01\x02\x03\x00-\x10\x20\x30\x7f-\x10\x20\x30\xff-',
+            (0, 0, 0, 0), (32, 64, 96, 127), (16, 32, 48, 255))
+        self.assert_unpack("RGBA", "RGBaXX",
+            b'\x01\x02\x03\x00==\x10\x20\x30\x7f!!\x10\x20\x30\xff??',
+            (0, 0, 0, 0), (32, 64, 96, 127), (16, 32, 48, 255))
```

Comment:
Just so I understand, why add the extra tuple for these tests?

Reply:
In RGBa → RGBA conversion alpha = 0 is special case and the result is undefined, which is 0 in UINT8 representation :~)

alpha = 255 is also special case, because x * 255 / 255 = x.

I have added something in the middle to be sure than RGB is actually divided by alpha: 16 * 255 / 127 ≈ 32